detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/TrimMultilineRawString.kt

```diff
@@ -0,0 +1,74 @@
+package io.gitlab.arturbosch.detekt.rules.style
+
+import io.gitlab.arturbosch.detekt.api.CodeSmell
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.api.Debt
+import io.gitlab.arturbosch.detekt.api.Entity
+import io.gitlab.arturbosch.detekt.api.Issue
+import io.gitlab.arturbosch.detekt.api.Rule
+import io.gitlab.arturbosch.detekt.api.Severity
+import org.jetbrains.kotlin.psi.KtCallExpression
+import org.jetbrains.kotlin.psi.KtExpression
+import org.jetbrains.kotlin.psi.KtStringTemplateExpression
+import org.jetbrains.kotlin.psi.psiUtil.getQualifiedExpressionForReceiver
+import org.jetbrains.kotlin.psi.psiUtil.getQualifiedExpressionForSelectorOrThis
+
+/**
+ * All the Raw strings that have more than one line should be followed by `trimMargin()` or `trimIndent()`.
+ *
+ * <noncompliant>
+ * """
+ *   Hello World!
+ *   How are you?
+ * """
+ * </noncompliant>
+ *
+ * <compliant>
+ * """
+ *   |  Hello World!
+ *   |  How are you?
+ * """.trimMargin()
+ *
+ * """
+ *   Hello World!
+ *   How are you?
+ * """.trimIndent()
+ *
+ * """Hello World! How are you?"""
+ * </compliant>
+ */
+class TrimMultilineRawString(val config: Config) : Rule(config) {
+    override val issue = Issue(
+        javaClass.simpleName,
+        Severity.Style,
+        "Multiline raw strings should be followed by `trimMargin()` or `trimIndent()`.",
+        Debt.FIVE_MINS
+    )
+
+    override fun visitStringTemplateExpression(expression: KtStringTemplateExpression) {
+        super.visitStringTemplateExpression(expression)
+
+        if (expression.text.lines().count() == 1) return
```

Comment:
What happens if the text of the expression is empty? Would this statement then yield 0?
Should the check be more conservative and flag `<= 1`?

```suggestion
        if (expression.text.lines().count() <= 1) return
```

Reply:
The expression should contain at least the `""` or `""""""`. But I agree the `<=` is safer.